ci/scripts/unit-tests

```diff
@@ -4,5 +4,5 @@ set -euo pipefail
 
 cd ${REPO_ROOT}
 
-bundle package
+bundle update
```

Comment:
Shouldn't this be bundle install, so it installs the versions from the gemfile and not what is the most current available?

Reply:
Yes we have to use `bundle install.` See help for install vs update:
For update: `Update the gems specified (all gems, if --all flag is used), ignoring the previously installed gems specified in the Gemfile.lock. In general, you should use bundle install(1) bundle-install.1.html to install the same exact gems and versions across machines.` But we want to install pinned versions.